lib/3.6/paths.cf

```diff
@@ -194,13 +194,21 @@ bundle common paths
       "path[grep]"     string => "/usr/bin/grep";
       "path[ls]"       string => "/bin/ls";
       "path[netstat]"  string => "/usr/bin/netstat";
-      "path[ping]"     string => "/usr/bin/ping";
       "path[perl]"     string => "/usr/bin/perl";
       "path[printf]"   string => "/usr/bin/printf";
       "path[sed]"      string => "/usr/bin/sed";
       "path[sort]"     string => "/usr/bin/sort";
       "path[tr]"       string => "/usr/bin/tr";
 
+    freebsd.!(freebsd_9_3|freebsd_10|freebsd_11)|netbsd|openbsd::
+  
+      "path[ping]"     string => "/usr/bin/ping";
+  
+    freebsd_9_3|freebsd_10|freebsd_11::
+  
+      "path[ping]"     string => "/sbin/ping";
+      "path[sysctl]"   string => "/sbin/sysctl";
```

Comment:
Accidental omission on 3.7, deliberate omission on 3.5. "path[sysctl]" is of course present, but I am planning to use it for future inventory work that depends on 3.6+ features. Corrected 3.7 so it's there. :)